client/src/components/map.tsx

```diff
@@ -112,8 +123,38 @@ const MapWrapper = ({features}: IMapWrapperProps) => {
     }
   }, [features]);
 
+  const handleMapClick = (event: { pixel: any; }) => {
+    // get clicked coordinate using mapRef to access current React state inside OpenLayers callback
+    //  https://stackoverflow.com/a/60643670
+    const clickedCoord = mapRef.current.getCoordinateFromPixel(event.pixel);
+
+    // transform coord to EPSG 4326 standard Lat Long
+    const transformedCoord = transform(clickedCoord, 'EPSG:3857', 'EPSG:4326');
+
+    // set React state
+    setSelectedCoord( transformedCoord );
+  };
+
+  const handlePointerMove = (event: { pixel: any; }) => {
+    if (selectedFeature !== undefined) {
+      selectedFeature?.setStyle(undefined);
+      setSelectedFeature(undefined);
+    }
+
+    mapRef.current.forEachFeatureAtPixel(event.pixel, (feature) => {
+      setSelectedFeature(feature);
+      return true;
+    });
+  };
+
   return (
-    <div ref={mapElement} className={styles.mapContainer}></div>
+    <>
```

Comment:
Is this tag supposed to be empty like this

Reply:
Yep this is shorthand for a React [fragment](https://reactjs.org/docs/fragments.html)